avm/res/synapse/workspace/administrators/main.bicep

```diff
@@ -0,0 +1,51 @@
+metadata name = 'Synapse Workspaces Administrators'
+metadata description = 'This module deploys Synapse Workspaces Administrators.'
+metadata owner = 'Azure/module-maintainers'
+
+@description('Conditional. The name of the parent Synapse Workspace. Required if the template is used in a standalone deployment.')
+param workspaceName string
+
+@description('Required. The administrators definition.')
+param administrator adminType
+
+resource workspace 'Microsoft.Synapse/workspaces@2021-06-01' existing = {
+  name: workspaceName
+}
+
+resource synapse_workspace_administrator 'Microsoft.Synapse/workspaces/administrators@2021-06-01' = if (administrator.login != '') {
```

Comment:
Hey @elanzel, should this condition be in the parent invocation rather than the child? If done in the parent, you woulnd't even run the deployment.
Also there is an `empty()` function you could use :)